test/e2e/apps/disruption.go

```diff
@@ -215,6 +201,38 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 		})
 	}
+
+	var _ = Describe("UpdatingPodDisruptionBudgets", func() {
+		It("Allows updates to pdb's", func() {
+			By("Creating a pdb that targets all three pods in a test replica set")
+			createPDBMinAvailableOrDie(cs, ns, intstr.FromInt(3))
+			createReplicaSetOrDie(cs, ns, 3, false)
+			waitForPodsOrDie(cs, ns, 3)
+
+			By("First trying to evict a pod which shouldn't be evictable")
+			var pod v1.Pod
```

Comment:
This line isn't needed.

Reply:
Ok, how about I move it above the test for the successful eviction instead?  That way I am ensuring that the pods are in the correct state before I attempt the successful eviction.